activity.py

```diff
@@ -183,6 +183,11 @@ def build_colors_toolbar(self, toolbox):
         # select color
         item = Gtk.ToolItem()
         _fill_color = ColorToolButton()
+        c = Gdk.Color(
```

Comment:
You should name the variable color.

Reply:
I used `c` because it is used in [others](https://github.com/sugarlabs/reversi-activity/blob/5d48e6c53d5a09a97a7b4697bc1fa359304e204e/activity.py#L109) too. Like for player colors and lines. I thought changing the variable name every where might just be extra diff....

Reply:
It's not extra diff.

Reply:
Done in a6e456f